src/hammer-vlsi/technology/sky130/defaults.yml

```diff
@@ -0,0 +1,84 @@
+# Settings for the sky130 technology to be overriden by the project.
+technology.sky130:
+  sky130_pdk: "PATH-TO-SKY130_PDK"
+  sky130_nda: "PATH-TO-skywater-src-nda"
+  sky130A: "PATH-TO-SKY130A_DIR"
+  openram_lib: "PATH-TO-sky130_fd_bd_sram/macros"
+  dffram_lib: "PATH-TO-DFFRAM/Compiler/build"
+
+  # GDS map file path, not provided by efabless
+  gds_map_file: "hammer/src/hammer-vlsi/technology/sky130/sky130_lefpin.map"
+  
+  lvs_deck_sources: 
+    - "${technology.sky130.sky130_nda}/s8/V2.0.1/LVS/Calibre/lvsControlFile_s8"
+  lvs_deck_sources_meta: lazysubst
+
+# Calibre environment variables
+mentor.extra_env_vars:
+  - PDK_HOME: "${technology.sky130_nda}/s8/V2.0.1"
+
+# Set some defaults for this technology.
+vlsi:
+  # Technology dimension
+  core:
+    technology: sky130
+    node: 130
+    sram_generator_tool: "sram_compiler"
+    sram_generator_tool_path: ["${HAMMER_HOME}/src/hammer-vlsi/technology/sky130"]
+    sram_generator_tool_path_meta: "append"
+  inputs:
+    # Supply voltages.
+    # TODO: add ability to tie pin to net in Hammer Innovus plugin
+    supplies:
+      power:  [ {name: "VDD",  pin: "VDD"}, 
+                {name: "VPWR", pin: "VPWR", tie: "VDD"},
+                {name: "VPB",  pin: "VPB" , tie: "VDD"}]
+      ground: [ {name: "VSS",  pin: "VSS"},
+                {name: "VGND", pin: "VGND", tie: "VSS"},
+                {name: "VNB",  pin: "VNB" , tie: "VSS"}]
+      VDD: "1.8 V"
+      GND: "0 V"
+
+    # moved these to json
+    # dont_use_mode: append
+    # dont_use_list: ["*sdf*","sky130_fd_sc_hd__probe_p_8"] # Scan flops go haywire! Avoid them. 
+    
+    # mmmc corners config
+    mmmc_corners: [
+      {
+        name: "sky130_fd_sc_hd__ss_100C_1v60",
+        type: "setup",
+        voltage: "1.60 V",
+        temp: "100 C"
+      },
+      {
+        name: "sky130_fd_sc_hd__ff_n40C_1v95",
+        type: "hold",
+        voltage: "1.95 V",
+        temp: "-40 C"
+      }
```

Comment:
Most SRAMs are only characterized at the TT corner, but I thought MMMC only cared about setup/hold? Should I add the TT corner and mark its type as "extra"?

Reply:
Yeah, as long as everything is characterized at the TT corner, it can be added to the list of MMMC corners as an extra corner. It barely adds any analysis time and is most beneficial for power analysis.